ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/PreconditionChecker.java

```diff
@@ -57,7 +57,7 @@ public class PreconditionChecker implements Executable {
 
   private static final Logger LOG = LoggerFactory.getLogger(PreconditionChecker.class);
 
-  final ServiceContext serviceContext;
+  final Supplier<ServiceContext> serviceContextSupplier;
   final KsqlRestConfig restConfig;
   final Supplier<Map<String, String>> propertiesLoader;
   final KafkaTopicClient topicClient;
```

Comment:
can you make the same change for the topic client?

Reply:
done